index.html

```diff
@@ -109,87 +133,76 @@ <h2>Tonic</h2>
-                        <span><i class="fa fa-chevron-down "></i></span>
-                    </div>
-                    <ul>
-                        <li class="pull">
-                            <img class="logos" src="images/Ellipse01.svg">
-                            <span class="wording">javascript</span>
-                        </li>
-                        <li class="pull">
-                            <img class="logos" src="images/Ellipse02.svg">
-                            <span class="wording">html</span>
-                        </li>
-                        <li class="pull">
-                            <img class="logos" src="images/Ellipse03.svg">
-                            <span class="wording">css</span>
-                        </li>
-                    </ul>
-                    <div class="languages">
-                        <h3>Frameworks</h3>
-                        <span><i class="fa fa-chevron-right"></i></span>
-                    </div>
-                    <div class="languages">
-                        <h3>Skills</h3>
-                        <span><i class="fa fa-chevron-right"></i></span>
-                    </div>
+                        <div class="languages">
+                            <h3>Languaje</h3>
+                            <span><i class="fa fa-chevron-down "></i></span>
+                        </div>
+                        <ul>
+                            <li class="pull">
+                                <img class="logos" src="images/Ellipse01.svg">
+                                <span class="wording">javascript</span>
+                            </li>
+                            <li class="pull">
+                                <img class="logos" src="images/Ellipse02.svg">
+                                <span class="wording">html</span>
+                            </li>
+                            <li class="pull">
+                                <img class="logos" src="images/Ellipse03.svg">
+                                <span class="wording">css</span>
+                            </li>
+                        </ul>
+                        <div class="languages">
+                            <h3>Frameworks</h3>
+                            <span><i class="fa fa-chevron-right"></i></span>
+                        </div>
+                        <div class="languages">
+                            <h3>Skills</h3>
+                            <span><i class="fa fa-chevron-right"></i></span>
+                        </div>
                 </div>
-   </section>
+        </section>
+    </main>
+    <footer>
+        <div class="bottom">
+            <h2 class="white">Contact me </h2>
+            <p class="#">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/jlberbessi@gmail.com"
+            method="POST">
```

Comment:
- [ ] Great job here 💯, but you should create a free account on [Formspree](https://formspree.io/) and choose New Form then Update your form's action. 
You have to replace "jlberbessi@gmail.com" with the unique "form_id" which you will get after creating the account and form on [Formspree](https://formspree.io/) 👌

![image](https://user-images.githubusercontent.com/45231276/193523287-b87766c2-a60b-4c6a-b519-90b762a16d56.png)

Reply:
Done. It was already set up. Thanks a lot for your recommendation.